__tests__/app.test.js

```diff
@@ -531,3 +531,60 @@ describe("DELETE /api/comments/:comment_id", () => {
     
   });
 });
+
+describe("GET /api/users", () => {
+  it("200: should respond with a 200 status code ", () => {
+    return request(app).get("/api/users").expect(200);
+  });
+
```

Comment:
You don't need to establish this here if you're also checking on it later on (like on line 551)